packages/DynamicHyperlinkTransformer/src/DynamicHyperlinkTransformer.js

```diff
@@ -1,25 +1,36 @@
 import React from "react";
 import PropTypes from "prop-types";
 import useI18n from "@paprika/l10n/lib/useI18n";
-import "./DynamicHyperlink.scss";
+import "./DynamicHyperlinkTransformer.scss";
 
 function isPromise(obj) {
   return obj && obj.then && typeof obj.then === "function";
 }
 
-export default function DynamicHyperlink({ onFetch }) {
+export default function DynamicHyperlinkTransformer({ onFetch }) {
   const I18n = useI18n();
 
   React.useEffect(() => {
     function updateDynamicHyperlinks() {
-      const dynamicHyperlinks = Array.from(document.querySelectorAll("[data-dynamic-hyperlink]"));
+      const dynamicHyperlinkSelector = "[data-dynamic-hyperlink]";
+      const dynamicHyperlinks = Array.from(document.querySelectorAll(dynamicHyperlinkSelector));
+
+      document.querySelectorAll("iframe.cke_wysiwyg_frame").forEach(iframe => {
+        if (iframe.contentWindow.document.body) {
+          iframe.contentWindow.document.body.querySelectorAll(dynamicHyperlinkSelector).forEach(dynamicHyperlink => {
+            dynamicHyperlinks.push(dynamicHyperlink);
+          });
+        } else {
+          setTimeout(updateDynamicHyperlinks, 100);
```

Comment:
is this timeout just to ensure that the iframe content is loaded before we load the links, is there a more reliable way than using timeouts?

Reply:
Yeah i don't like it either, but without it, it will not work on links inside ckEditor.
I don't think there is a better way to do it since the ckEditor is not React.  @nahumzs do you have any thoughts?

Reply:
Wouldn't the first iframe be skipped since it'll just go to the else case and never back to the if case? Perhaps need a while loop with condition `!iframe.contentWindow.document.body` to wait for the content to be loaded.

Reply:
I answered a similar question here https://stackoverflow.com/a/44979019/196038

I think that should be enough to just wait until `iframe.contentWindow.document.body.querySelectorAll(dynamicHyperlinkSelector)` return something. 

By the way, in that example, I received a string as a DOM parameter, but it should be a function instead, so the query always gets executed with the current context after each `requestNextAnimationFrame()`